scientific-computing-with-python/learn-numerical-analysis-by-finding-the-square-root-of-a-number-using-the-bisection-method/bisection-method.py

```diff
@@ -0,0 +1,34 @@
+def square_root_bisection(square_target, tolerance=1e-7, max_iterations=100):
+    if square_target < 0:
+        # Square root of negative number is not defined in real numbers
+        raise ValueError(
+            'Square root of negative number is not defined in real numbers')
+    if square_target == 1:
+        return 1  # Shortcut for square root of 1
```

Comment:
```suggestion
    if square_target == 1:
        return 1  # Shortcut for square root of 1
    elif square_target == 0:
        return 0  # Shortcut for square root of 0
```

Seeing `The square root of 0 is approximately 0.000244140625` is terribly annoying me.

Reply:
I realized nothing was being printed for` 0` and `1`. I have added explicit print statements for each and also improved the alignment of the code.